receiver/riakreceiver/scraper_test.go

```diff
@@ -156,3 +193,26 @@ func TestScaperScrape(t *testing.T) {
 		})
 	}
 }
+
+func customMetricSettings() metadata.MetricsSettings {
+	return metadata.MetricsSettings{
+		RiakMemoryLimit: metadata.MetricSettings{
+			Enabled: false,
+		},
+		RiakNodeOperationCount: metadata.MetricSettings{
+			Enabled: false,
+		},
+		RiakNodeOperationTimeMean: metadata.MetricSettings{
+			Enabled: true,
+		},
+		RiakNodeReadRepairCount: metadata.MetricSettings{
+			Enabled: true,
+		},
+		RiakVnodeIndexOperationCount: metadata.MetricSettings{
+			Enabled: true,
+		},
+		RiakVnodeOperationCount: metadata.MetricSettings{
+			Enabled: true,
+		},
+	}
+}
```

Comment:
This is creating a specific scenario. Now that we have a `setupCfg` setting for each test, just do this in the specific test that requires this.